RAGchain/preprocess/text_splitter/token_splitter.py

```diff
@@ -0,0 +1,103 @@
+import copy
+
+from typing import List
+from uuid import uuid4
+
+from langchain.schema import Document
+from langchain.text_splitter import CharacterTextSplitter, TokenTextSplitter, SpacyTextSplitter, SentenceTransformersTokenTextSplitter, NLTKTextSplitter
+from transformers import GPT2TokenizerFast
+from RAGchain.preprocess.text_splitter.base import BaseTextSplitter
+from RAGchain.schema import Passage
+
+
+class Token_Splitter(BaseTextSplitter):
+    """
+    Split a document into passages by recursively splitting on a list of separators.
+    You can specify a window_size and overlap_size to split the document into overlapping passages.
+    """
+    def __init__(self, tokenizer_name: str = 'tiktoken', chunk_size: int = 0, chunk_overlap: int = 0, **kwargs):
```

Comment:
Default chunck_size is 0???
I think that is too small, isn't it?

Reply:
I agree.